Tests/Engine/InvokeScriptAnalyzer.tests.ps1

```diff
@@ -397,6 +397,18 @@ Describe "Test CustomizedRulePath" {
                 }
             }
 
+            It "Should process relative settings path even when settings path object is an expression" {
+                try {
+                    $initialLocation = Get-Location
+                    Set-Location $PSScriptRoot
```

Comment:
this should probably be `Push-Location`, that way you don't need to save the location at all.